<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serveurwidget</class>
 <widget class="QWidget" name="Serveurwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveurwidget</string>
  </property>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Num√©ro de port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clients</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPort">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65000</number>
   </property>
   <property name="value">
    <number>1025</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLancement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancement serveur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Serveurwidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
